packages/ecs-gpu-init/ecs-gpu-init.service

```
@@ -0,0 +1,13 @@
+[Unit]
+Description=Initialize ECS agent GPUs config
```

nit: maybe simplify a bit:
```suggestion
Description=Initialize ECS GPU config
```